Description: Fix for building on IA64
Author: John Paul Adrian Glaubitz
Last-Updated: 2020-06-09
Forwarded: yes
Bug-Origin: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=963729


--- a/opal/include/opal/sys/timer.h
+++ b/opal/include/opal/sys/timer.h
@@ -78,8 +78,6 @@
 #include "opal/sys/arm64/timer.h"
 #elif OPAL_ASSEMBLY_ARCH == OPAL_IA32
 #include "opal/sys/ia32/timer.h"
-#elif OPAL_ASSEMBLY_ARCH == OPAL_IA64
-#include "opal/sys/ia64/timer.h"
 #elif OPAL_ASSEMBLY_ARCH == OPAL_POWERPC32
 #include "opal/sys/powerpc/timer.h"
 #elif OPAL_ASSEMBLY_ARCH == OPAL_POWERPC64
