Description: Modify arm "detection" for Raspbian.

Index: openmpi-2.1.1.try3/config/opal_config_asm.m4
===================================================================
--- openmpi-2.1.1.try3.orig/config/opal_config_asm.m4
+++ openmpi-2.1.1.try3/config/opal_config_asm.m4
@@ -1037,7 +1037,7 @@ AC_DEFUN([OPAL_CONFIG_ASM],[
             OPAL_GCC_INLINE_ASSIGN='"mov %0, #0" : "=&r"(ret)'
             ;;
 
-        armv7*|arm-*-linux-gnueabihf)
+        armv7*)
             opal_cv_asm_arch="ARM"
             OPAL_ASM_SUPPORT_64BIT=1
             OPAL_ASM_ARM_VERSION=7
@@ -1046,11 +1046,10 @@ AC_DEFUN([OPAL_CONFIG_ASM],[
             OPAL_GCC_INLINE_ASSIGN='"mov %0, #0" : "=&r"(ret)'
             ;;
 
-        armv6*)
+        armv6*|arm-*-linux-gnueabihf)
             opal_cv_asm_arch="ARM"
             OPAL_ASM_SUPPORT_64BIT=0
             OPAL_ASM_ARM_VERSION=6
-            CCASFLAGS="$CCASFLAGS -march=armv7-a"
             AC_DEFINE_UNQUOTED([OPAL_ASM_ARM_VERSION], [$OPAL_ASM_ARM_VERSION],
                                [What ARM assembly version to use])
             OPAL_GCC_INLINE_ASSIGN='"mov %0, #0" : "=&r"(ret)'
